pkg/sctx/sctx.go

```diff
@@ -67,6 +67,14 @@ func GetGasLimit(ctx context.Context) uint64 {
 	return 0
 }
 
+func GetGasLimitWithDefault(ctx context.Context, defaultLimit uint64) uint64 {
```

Comment:
would having similar method for getting gas price (with default value) would also be helpful?